### DISCUSSION TITLE
How can I disable 4 sec delay highlight

### DISCUSSION THREAD

**Discussion by khangal** [nvim-treesitter/nvim-treesitter#2719] [Q&A]
If i don't move my cursor 4-5 seconds it highlights area, it make hard to read big multiline strings.

Example highlight
![Screen Shot 2022-04-02 at 13 51 43](https://user-images.githubusercontent.com/10184641/161368815-dda8ca16-c5ad-4cb8-9ce6-3fa9f2cc2ff6.png)

**Comment by stsewd** [comment]
You are probably using this plugin https://github.com/nvim-treesitter/nvim-treesitter-refactor#highlight-definitions, or another plugin that does that, nvim-treesitter doesn't do that by default.
